### ISSUE TITLE
A11y_AzureToolsStorageExplorer_UploadFolder_ScreenReader:NVDA announces incorrect name for the "Browse folder(...)" button.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/AzureStorageExplorer#2608]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier."
​

**GitHubTags:** #A11y_AzureToolsStorageExplorer; #A11YTCS; #DesktopApp; #MAS1.3.1; #MAS4.1.2; #NVDA; #JAWS; #Win32; #Benchmark; #A11ySev2;

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.12.0
Windows10
NVDA version: 2019.2.1

**Repro Steps**:
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Navigate to "Upload" button and hit enter on  "Upload Folder" control.
6. Start the NVDA screen reader.
7. Navigate to the (...) browse folder button.
8. Observe the NVDA announcement.

**Actual:**
NVDA announces the incorrect name for the "Browse folder(...)" button.Here announces as "Browse files".

**Expected:**
NVDA should announces the correct name for the "browse folder(...)" button.It should announces like "browse folder".

**User Impact:**
Screen reader user get incorrect information regarding the control if it announces the incorrect name for the "Browse folder" control.

**Recommendation:**
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
**MAS1.3.1:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**MAS4.1.2:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Comment by Suraj1200499** [comment]
We have uploaded repro video of this bug in OneDrive link with name "2608_A11y_AzureToolsStorageExplorer_UploadFolder_ScreenReader" please refer the Screenshot for reference.
Please let us know if there is any problem in accessing the link

**Comment by JasonYeMSFT** [comment]
@vinodbk197 Could you please verify and close this issue if it is fixed?

**Comment by pranshu512** [comment]
I verified this issue on Version 1.16.0-dev
[2608_Closed.zip](https://github.com/microsoft/AzureStorageExplorer/files/5347592/2608_Closed.zip)
 and found that it is not reproducing .
Refer attachment
